doc/en/warnings.rst

```diff
@@ -276,7 +276,7 @@ argument ``match`` to assert that the exception matches a text or regex::
     ...     warnings.warn("this is not here", UserWarning)
     Traceback (most recent call last):
       ...
-    Failed: DID NOT WARN. No warnings of type ...UserWarning... was emitted...
+    _pytest.outcomes.Failed: DID NOT WARN. No warnings of type ...UserWarning... was emitted...
```

Comment:
maybe `__module__ = 'pytest'` -- I think we do this elsehwere (assuming this is an exported name)

Reply:
See above, it is not importable like that.

Reply:
shall we make it so?

Reply:
See above, feel free - closing.